### DISCUSSION TITLE
how to change custom status with discord js ?

### DISCUSSION THREAD

**Discussion by Devabdellah** [marioparaschiv/nitro-sniper#165] [Q&A]
i looked but everyone is changing the activity
I don't want the watching , playing... 
I'm talking about this : https://support.discord.com/hc/en-us/articles/360035407531-Custom-Status

**Comment by Devabdellah** [comment]
@slow @anyone xd

**Comment by Devabdellah** [comment]
:/

**Comment by accomplis** [comment]
i made a program for that in python for running on repl
